### PR TITLE
Update game 2

### PR DESCRIPTION
**Pull request by JhengCaldona**
this is the second time

### CODE REVIEW

**Review by JhengCaldona — COMMENTED** [review]
sample comment on update game 2

File: README.md [hunk 1]
```diff
@@ -14,51 +14,30 @@ _Collaborate and work together on GitHub._
...
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 4 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
```
Comment (caldonaje) [inline README.md:22]: sample comment

File: README.md [hunk 2]
```diff
@@ -14,51 +14,30 @@ _Collaborate and work together on GitHub._
...
 
-- **Who is this for**: Developers, new GitHub users, users new to Git, students, managers, teams.
-- **What you'll learn**: When and how to request a review; how to provide a review of someone else's pull request.
-- **What you'll build**: We'll be reviewing a pull request for a simple game.
-- **Prerequisites**: We assume you are familiar with creating branches, commits, and pull requests—you can learn this in our [Introduction to GitHub](https://github.com/skills/introduction-to-github) course.
-- **How long**: This course takes less than 30 minutes to complete.
+Now that you have explored the different ways you can review a pull request it is time to learn how to use _suggest changes_.
 
-In this course, you will:
+**What is _suggest changes_?**: This feature enables you to recommend a change to a pull request that the author can commit with the push of a button.
```
Comment (caldonaje) [inline README.md]: sample comment

File: README.md (hunk 1 above)
Comment (caldonaje) [reply README.md:22]: ok

**Review by JhengCaldona — COMMENTED** [review]
fiinish na

**Pull request merged**